direct/core/src/main/java/cz/o2/proxima/direct/transaction/TransactionalOnlineAttributeWriter.java

```diff
@@ -171,9 +180,41 @@ public void commitWrite(List<StreamElement> outputs, CommitCallback callback)
       writer.write(toWrite, compositeCallback);
     }
 
```

Comment:
NIT: One missed smell from previous: `TransactionRejectedException` should be probably static

Reply:
:+1: